### ISSUE TITLE
Allow Overriding All Styles

### ISSUE THREAD

**Issue by boyter** [boyter/searchcode-server#107]
Add the ability to override all styles. This will mean changing the code to allow http://sparkjava.com/documentation#static-files static files by path. Only catch is that this requires a reboot to take effect.

**Comment by boyter** [comment]
This has been partially implemented through the ability to embed custom HTML/CSS/JS on all pages through the settings screen for non community versions.

**Comment by boyter** [comment]
The problem with this is that you cannot easily change the location of the templates. Need to investigate this further.

**Comment by quasarea** [comment]
Could you provide some description/example how to use this feature?

**Comment by boyter** [comment]
If you mean embedding the custom HTML/JS this is not supported in the community edition. You would have to compile your own version with the community flag off to enable it.

If you mean for full theming, this is not yet supported at all. I am still investigating on how to achieve this.

**Comment by quasarea** [comment]
I do compile on my own this days :) so should it be something like embed.html file in certain folder or smth?

**Comment by boyter** [comment]
No. Its a setting on the settings page in the admin section. You can embed your own javascript/css/html there.

**Comment by boyter** [comment]
https://github.com/perwendel/spark/issues/892#issuecomment-319286836
